components/brave_extension/extension/brave_extension/background/events/cosmeticFilterEvents.ts

```diff
@@ -82,6 +109,14 @@ export function onContextMenuClicked (info: chrome.contextMenus.OnClickData, tab
     case 'manageCustomFilters':
       openFilterManagementPage()
       break
+    case 'elementPickerMode': {
+      chrome.tabs.query({ active: true, currentWindow: true }, (tabs: [chrome.tabs.Tab]) => {
+        if (tabs.length > 0) {
```

Comment:
Under what condition would this hold?  A comment would help to make sure we're not ignoring an important error case

Reply:
this means chromium can't find the active tab. it's a safe assumption to make in the extension, and we do it in other parts of the code too.